errors/public.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package errors
+
+import (
+	"errors"
+	"fmt"
+)
+
+// UPGRADEv2: delete other files in folder.
+//nolint
+var (
+	ErrRunFailure      = errors.New("cannot run check")
+	ErrRepoUnreachable = errors.New("repo unreachable")
```

Comment:
This is same as this one: https://github.com/ossf/scorecard/blob/main/clients/repo_client.go#L21. Can we put a TODO to remove the `repo_client.go` error?

Reply:
done.